plugins/source/aws/docs/tables/aws_kms_grants.md

```diff
@@ -0,0 +1,27 @@
+# Table: aws_kms_grants
+
+https://docs.aws.amazon.com/kms/latest/APIReference/API_GrantListEntry.html
+
+The composite primary key for this table is (**account_id**, **region**, **grant_id**, **key_id**).
+
+## Relations
+This table depends on [aws_kms_keys](aws_kms_keys.md).
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|account_id (PK)|String|
```

Comment:
Will the account_id always match the issuing account?

Reply:
@jsonpr Yes that is assumed, it's resolved from the `Client` and since we use the key ID in the `ListGrants` call...